README.md

```diff
@@ -48,10 +48,6 @@ You need to setup [Yaml configuration files](docs/Configuration.md) to specify y
 
 There are also some [advanced configuration options](docs/AdvancedFeatures.md) supported such as loops, conditional statements and permissions for anonymous.
 
-# User Interface
-
-There is a Felix Web Console plugin (at `/system/console/actool`) as well as a Touch UI console (at `/mnt/overlay/netcentric/actool/content/overview.html`) to apply configurations and to inspect previous executions of the tool. Additionally there is a [JMX interface](docs/Jmx.md) for some advanced use cases.
-
```

Comment:
Maybe we should leave this documentation in (or just adjust it)?

Reply:
I don’t like duplication. This is described in the other md file

Reply:
I restored it and linked out to the other page now.